app/boards/shields/tg4x/Kconfig.defconfig

```
@@ -4,6 +4,6 @@
 if SHIELD_TG4X
 
 config ZMK_KEYBOARD_NAME
-    default "TG4X"
+    default "tg4x"
```

```suggestion
    default "TG4x"
```
The silkscreen on the board shows this style of capitalization.